sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableConfig.java

```diff
@@ -179,12 +179,8 @@ void populateDisplayData(DisplayData.Builder builder) {
 
     if (getBigtableOptions() != null) {
       builder.add(DisplayData.item("bigtableOptions", getBigtableOptions().toString())
-        .withLabel("Bigtable Options"));
+        .withLabel("Provided Bigtable Options"));
```

Comment:
Are you trying to differentiate options given by the user and the "effective" options?

Reply:
Not anymore, before we had those here. Fixed.